packages/task/src/browser/task-configurations.ts

```diff
@@ -54,7 +54,7 @@ export class TaskConfigurations implements Disposable {
     /**
      * Map of source (path of root folder that the task configs come from) and task customizations map.
      */
-    protected taskCustomizationMap = new Map<string, TaskCustomization[]>();
+    protected taskCustomizationMap = new Map<string, TaskConfiguration[]>();
```

Comment:
Question to reviewers:

- Is it safe to change this to `TaskConfiguration`
- See the code below, it seems the `_scope` property would always exist
  and another branch seems to assume the label property also exists.
- https://github.com/eclipse-theia/theia/pull/9501/files#diff-a4fc4b67b6c99119f65bcbfd083563d1a18a56d07410a7578d06b767c81697caR402-R406